train.py

```diff
@@ -59,11 +59,11 @@ def get_base_argument_parser():
     # logging
     parser.add_argument("--experiment_name", type=str, default="dev")
     parser.add_argument("--logs_path", type=str, default="log")
+    parser.add_argument("--save_k_top_val_models", type=int, default=1)
 
     parser.add_argument("--filepath_to_arguments_json_config", type=str, default=None)
 
-    parser.add_argument("--save", dest="save", default=True, action="store_true")
-    parser.add_argument("--nosave", dest="save", default=True, action="store_false")
+    parser.add_argument("--save_top_n_val_models", type=int, default=1)
```

Comment:
Why are there two separate arguments that seem to do the same thing here?

Reply:
previously the train script had `--save` and also  `--nosave` - I presumed this was you, so replaced it with a single arg.

Reply:
Also that was not me. It was there originally.